kratos/processes/fast_transfer_between_model_parts_process.cpp

```diff
@@ -161,7 +167,7 @@ void FastTransferBetweenModelPartsProcess::TransferWithFlags()
 
         if (num_constraints != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::CONSTRAINTS || mEntity == EntityTransfered::NODESANDCONSTRAINTS)) {
             const auto it_const_begin = mrOriginModelPart.MasterSlaveConstraintsBegin();
-            #pragma omp for
+            #pragma omp for schedule(guided, 512)
```

Comment:
```suggestion
            #pragma omp for
```

this is not supported by the parallel utilities, hence better remove it

I assume you only copy pasted this?

Reply:
Look that this is inside a parallel block and there is a critical at the end, I cannot use parallel_utils

Reply:
I am not talking about the parallel, only about the scheduling clause you add

Reply:
BTW parallel utils can be used but rewriting this is a bit of an effort (and not for this PR)

Reply:
I added one because I added one more block ahead (and the only one without scheduling must be the last one)

Reply:
(The final removal will be done with the parallel utilities refactoring)